### PR TITLE
Update integrations tests

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/AutoReview/FixerFactoryTest.php [hunk 1]
```diff
@@ -153,7 +153,6 @@ public function provideFixersPriorityCases()
             array($fixers['phpdoc_summary'], $fixers['phpdoc_trim']),
             array($fixers['phpdoc_to_comment'], $fixers['no_empty_comment']),
             array($fixers['phpdoc_to_comment'], $fixers['phpdoc_no_useless_inheritdoc']),
-            array($fixers['phpdoc_var_without_name'], $fixers['phpdoc_trim']),
```
Comment (SpacePossum) [inline tests/AutoReview/FixerFactoryTest.php]: there is no priority issue between these fixers

**Comment by julienfalque** [comment]
Please rebase on 2.12.

**Comment by SpacePossum** [comment]
we're mot doning a 2.2.21?

**Comment by julienfalque** [comment]
Nope: https://medium.com/@keradus/to-make-a-difference-8dce5f7cf9bf

Though branch 2.12 doesn't exist yet, I guess master is fine for now.

**Comment by SpacePossum** [comment]
resolved conflicts after target branch was changed, forced pushed

**Pull request merged**